tesseract.py

```diff
@@ -4,21 +4,20 @@
 import numpy as np
 import colorsys
 
-os.environ["SDL_VIDEO_CENTERED"] = '1'
+os.environ["SDL_VIDEO_CENTERED"] = "1"
```

Comment:
This edit is personal taste, normal strings should be double quotes and anything else (f, b, and r strings) should use single quotes.